bankaccount/client.go

```diff
@@ -28,13 +35,7 @@ func (c Client) New(params *stripe.BankAccountParams) (*stripe.BankAccount, erro
 	params.AppendToAsSourceOrExternalAccount(body)
 
 	ba := &stripe.BankAccount{}
-	var err error
-	if params.Customer != nil {
-		err = c.B.Call("POST", stripe.FormatURLPath("/customers/%s/sources", stripe.StringValue(params.Customer)), c.Key, body, &params.Params, ba)
-	} else {
-		err = c.B.Call("POST", stripe.FormatURLPath("/accounts/%s/external_accounts", stripe.StringValue(params.Account)), c.Key, body, &params.Params, ba)
-	}
-
+	err := c.B.CallRaw("POST", path, c.Key, body, &params.Params, ba)
```

Comment:
> ack that sounds fine for now though it's likely worth trying to revisit in the future to make it easier to reason with. It's easy to miss that there are some edge-cases when you build a new module or revamp this one.

Yeah, totally agree. I think with a little more effort we could get this reconciled into the standard framework.

> It might be worth a comment in the code though wdyt?

Yep, good call. I'll add that here and to the implementation in card as well (which uses a similar pattern).